inc/admin.php

```diff
@@ -1356,46 +1356,73 @@ public function get_layout_directories(){
 	public static function display_footer_premium_link() {
 		$links = array(
 			array(
-				'text' => __( 'Get an Accordion Addon for SiteOrigin Widgets', 'siteorigin-panels' ),
-				'url' => SiteOrigin_Panels::premium_url( 'plugin/accordion' )
-			),			
+				'text' => __( 'Get the row, cell, and widget %link%.', 'siteorigin-panels' ),
+				'url' => SiteOrigin_Panels::premium_url( 'plugin/animations' ),
+				'anchor' => __( 'Animations Addon', 'siteorigin-panels' ),
+			),
+			array(
+				'text' => __( 'Get the %link%. Build custom post types with reusable Page Builder layouts.', 'siteorigin-panels' ),
+				'url' => SiteOrigin_Panels::premium_url( 'plugin/cpt-builder' ),
+				'anchor' => __( 'CPT Builder Addon', 'siteorigin-panels' ),
+			),
+			array(
+				'text' => __( 'Get the %link%. Add beautiful and customizable text overlays with animations to your images.', 'siteorigin-panels' ),
+				'url' => SiteOrigin_Panels::premium_url( 'plugin/image-overlay' ),
+				'anchor' => __( 'Image Overlay Addon', 'siteorigin-panels' ),
+			),
 			array(
-				'text' => __( 'Get the row, cell, and widget Animations Addon', 'siteorigin-panels' ),
-				'url' => SiteOrigin_Panels::premium_url( 'plugin/animations' )
+				'text' => __( 'Get a %link% for SiteOrigin Image, Masonry, and Slider Widgets.', 'siteorigin-panels' ),
```

Comment:
Should this include _the_?

`Get a %link% for the SiteOrigin Image, Masonry, and Slider Widgets.`